### PR TITLE
Update definition page copy

### PR DESCRIPTION
**Pull request by katjam**
Trello card: https://trello.com/c/B0ClLPYb/371-master-copy-for-beta-01

## Description

- Definition page copy
- Keywords wrapped in [] will be styled in later PR

@neontribe/sea-map

### CODE REVIEW

File: src/Copy/Keys.elm [hunk 1]
```diff
@@ -68,38 +68,32 @@ type Key
     | DefinitionConciseP1
     | DefinitionConciseP2
     | DefinitionConciseP3
-    | DefinitionMoreLink
+    | SplitterAffirmation
```
Comment (Nikomus) [inline src/Copy/Keys.elm:71]: Is this still a splitter affirmation?

File: src/Copy/Text.elm [hunk 2]
```diff
@@ -188,113 +188,95 @@ t key =
...
+            "We had a joint account but he never paid anything into it. All the bills came out, even the repayments for the car. But it was in his name, not mine. And he wouldn't let me drive it without begging."
 
         DefinitionCategory3Title ->
-            "Sabotaging your finances and economic resources"
+            "Destroying your resources and economic security"
 
         DefinitionCategory3Info ->
-            "An abuser may try to sabotage your finances or whatever economic resources you do have. They might steal money or property from you or may cause damage to your property to devalue the house. Abusers might spend the money set aside for savings or holidays or bills or force all credit cards and bills to be in your name."
+            "An abuser may try to weaken or sabotage your finances or whatever economic resources you do have. This could mean [stealing money or property] from you or damaging your house. It could also mean spending the money set aside for [savings, holidays or bills]. They could also force all credit cards and bills to be in your name, so they don't build up debt in their name."
```
Comment (Nikomus) [inline src/Copy/Text.elm:236]: sabotage still here..

**Review by Nikomus — APPROVED** [review]
Looks OK, but a couple Qs 🌍

File: src/Copy/Text.elm (hunk 2 above)
Comment (katjam) [reply src/Copy/Text.elm:236]: Yes, sadly

File: src/Copy/Keys.elm (hunk 1 above)
Comment (katjam) [reply src/Copy/Keys.elm:71]: I'm hoping it will move back down, right now it's just affirmation

**Pull request merged**